src/main/java/frc/robot/TeleopInput.java

```diff
@@ -2,6 +2,7 @@
 
 // WPILib Imports
 import edu.wpi.first.wpilibj.Joystick;
+import edu.wpi.first.wpilibj.XboxController;
```

Comment:
Are we using the xbox controller for competition?

Reply:
yes